docs/90_partners.md

```diff
@@ -1,17 +1,36 @@
 # Project partners
 
-* Walther Blom, Dell Technologies, High Education & Research
-* Jaco van Dijk, Dell Technologies, Higher Education
-* Bob Dröge, Center for Information Technology, University of Groningen
-* Robbert Eggermont, Delft University of Technology
-* Kenneth Hoste, HPC-UGent
-* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS), University of Twente, Enschede
-* Caspar van Leeuwen, SURF
-* Koen Mulderij, Delft University of Technology
-* Alan O'Cais, Juelich Supercomputing Centre
-* Thomas Röblitz, University of Oslo
-* Mark Sharpley, University of Cambridge Research Computing Services Division
-* Peter Stol, Vrije Universiteit Amsterdam
-* Marco Verdicchio, SURF
-* Bas van der Vlies, SURF
-* Henk-Jan Zilverberg, Center for Information Technology, University of Groningen
+## [Delft University of Technology (The Netherlands)](https://www.tudelft.nl)
+* Robbert Eggermont
+* Koen Mulderij
+
+## [Dell Technologies (Europe)](https://www.dell.com/)
+* Walther Blom, High Education & Research
+* Jaco van Dijk, Higher Education
+
+## [Ghent University (Belgium)](https://www.ugent.be/hpc/en)
+* Kenneth Hoste
```

Comment:
```suggestion
* Kenneth Hoste (HPC-UGent)
```

(since a similar "subdepartment" is mentioned for others as well)

Reply:
It seems the other entries have ", ..." instead of "(...)"?